packages/@sanity/components/src/buttons/styles/DefaultButton.css

```diff
@@ -95,6 +95,10 @@
   display: flex;
   border-radius: 0;
   height: 100%;
+
+  @nest & .content {
+    margin: 0 !important;
```

Comment:
Hm, why do we need to use `!important` here?

Reply:
When making the small dropdown button i thought it was needed, turns out it was not for this case.